.buildkite/hooks/pre-command

```diff
@@ -41,7 +41,7 @@ if [[ "$BUILDKITE_PIPELINE_SLUG" == "beats" || "$BUILDKITE_PIPELINE_SLUG" == "fi
     git reset --quiet --hard
```

Comment:
It's not in scope, but it should be removed. Maybe raise a follow-up work. Leaving a comment so we don't forget about it.